batch/batch/front_end/templates/batch.html

```diff
@@ -44,7 +44,8 @@ <h2>Jobs</h2>
 	    <li>!term - jobs not matched by term</li>
 	  </ul>
 	</div>
-      </div>
+	  </div>
+	<div class='flex-col' style="overflow: scroll;">
```

Comment:
stretches to the width of the parent, so if parent would overflow, so would this container.